cli/git.go

```diff
@@ -34,6 +34,31 @@ type remoteApp struct {
 	Name    string
 }
 
+func gitRemoteNames() (results []string, err error) {
+	b, err := exec.Command("git", "remote").Output()
+	if err != nil {
+		return nil, err
+	}
+
+	s := bufio.NewScanner(bytes.NewBuffer(b))
+	s.Split(bufio.ScanWords)
+
+	results = make([]string, 1)
```

Comment:
This results in an empty string at the beginning of the slice, is that intentional? If not:

```go
var results []string
```